index.html

```diff
@@ -73,10 +73,21 @@ <h1 class="header-title-text">
         </div>
       </div>
       <div class="header-image-box">
-        <img class="header-image" src="https://picsum.photos/1000/500" />
-        <div class="button-border">
-          <div class="play"></div>
+        <video id="video-player" muted loop>
+          <source
+            src="resources/Video/Slaps Slow Motion.mp4"
```

Comment:
change url to lowercase and replace spaces with hyphens

Reply:
Done